config/ha.py

```diff
@@ -13,6 +13,22 @@
 
 NUMERALS = byte.DIGIT
 
+
+APOSTROPHE_TO_HOOK_Y = cdrewrite(
+    cross("ʼy", "ƴ"),
+    "",
+    "",
+    byte.BYTES.closure())
+
+HOOK_TO_APOSTROPHE_Y = cdrewrite(
+    cross("ƴ", "ʼy"),
+    "",
+    "",
+    byte.BYTES.closure())
+
+#LANGUAGE_SPECIFIC_PREPROCESSING = APOSTROPHE_TO_HOOK_Y
```

Comment:
Are we going to toggle these on and off?  You'll probably have to comment out the unused rewrite too if pylint complains that it's unused.

Reply:
Pylint doesn't appear to complain here, but I'll comment the unused rewrite out for now just to be safe, and just switch them out when I try the other experiment.

Reply:
SGTM!